contrib/auto-completion/packages.el

```diff
@@ -202,7 +202,7 @@
 
       ;; add key into candidate list
       (setq helm-yas-display-key-on-candidate t)
-      (setq spacemacs--snippets-dir
+      (setq spacemacs--auto-completion-dir
             (concat (ht-get configuration-layer-paths 'auto-completion)
```

Comment:
ok changing it now

On Mon, Jun 8, 2015 at 12:54 AM, Sylvain Benner <notifications@github.com>
wrote:

> In contrib/auto-completion/packages.el
> <https://github.com/syl20bnr/spacemacs/pull/1890#discussion_r31888258>:
>
> > @@ -202,7 +202,7 @@
> >
> >        ;; add key into candidate list
> >        (setq helm-yas-display-key-on-candidate t)
> > -      (setq spacemacs--snippets-dir
> > +      (setq spacemacs--auto-completion-dir
> >              (concat (ht-get configuration-layer-paths 'auto-completion)
>
> Missed this on the previous commit, you can use configuration-layers/get-layer-property
> 'auto-completion :dir which should return the absolute path of the layer
> (auto-completion directory included, I believe).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/syl20bnr/spacemacs/pull/1890/files#r31888258>.
>